Issue #2 Resolved by creating preprocessing scripts

mochiiten9158 opened the pull request.

yashkurkure commented:
@mochiiten9158 Take a look at the updates I made to `preprocessing/preprocessing_polaris_23_24.py`.

Since the paths on Windows and Linux work differently, I had to change how you read and write files. I changed it so that all input files are read from  `preprocessing/data` and all output files go to ` `preprocessing/output`

Further tasks:

- [ ] Print the statistics of each trace (take a look at the TODOs I left in `preprocessing/preprocessing_polaris_23_24.py`)
- [ ] Do the same changes I made but to `preprocessing/preprocessing_theta_23_24.py`

mochiiten9158 commented:
> @mochiiten9158 Take a look at the updates I made to `preprocessing/preprocessing_polaris_23_24.py`.
> 
> Since the paths on Windows and Linux work differently, I had to change how you read and write files. I changed it so that all input files are read from `preprocessing/data` and all output files go to `preprocessing/output`
> 
> ## Instructions on running preprocessing_polaris_23_24.py
> For the script `preprocessing_polaris_23_24.py` to run, download the input data from https://drive.google.com/file/d/1_W5-R6bzcTk4QXBRae56FJX-fbek_mLp/view?usp=drive_link and extract the files into `preprocessing/data`. Then from the directory `preprocessing/` run `python3 preprocessing_polaris_23_24.py` and it should create a new folder called `output` in the same directory with all the Polaris related files.
> 
> ## Further tasks:
> * [x]  Print the statistics of each trace (take a look at the TODOs I left in `preprocessing/preprocessing_polaris_23_24.py`)
> * [x]  Do the same changes I made but to `preprocessing/preprocessing_theta_23_24.py`



> @mochiiten9158 Take a look at the updates I made to `preprocessing/preprocessing_polaris_23_24.py`.
> 
> Since the paths on Windows and Linux work differently, I had to change how you read and write files. I changed it so that all input files are read from `preprocessing/data` and all output files go to `preprocessing/output`
> 
> ## Instructions on running preprocessing_polaris_23_24.py
> For the script `preprocessing_polaris_23_24.py` to run, download the input data from https://drive.google.com/file/d/1_W5-R6bzcTk4QXBRae56FJX-fbek_mLp/view?usp=drive_link and extract the files into `preprocessing/data`. Then from the directory `preprocessing/` run `python3 preprocessing_polaris_23_24.py` and it should create a new folder called `output` in the same directory with all the Polaris related files.
> 
> ## Further tasks:
> * [x]  Print the statistics of each trace (take a look at the TODOs I left in `preprocessing/preprocessing_polaris_23_24.py`)
> * [x]  Do the same changes I made but to `preprocessing/preprocessing_theta_23_24.py`

Finished above mentioned tasks, pushed changes to branch shamb.

yashkurkure commented:
gg
